x/evm/keeper/abci.go

```diff
@@ -31,6 +31,13 @@ func (k *Keeper) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
 	// reset counters that are used on CommitStateDB.Prepare
 	k.Bloom = big.NewInt(0)
 	k.TxCount = 0
+
+	//loading revision id from store in every block, make sure the `csdb.nextRevisionId` consistent in every node
+	id, err := k.GetRevisionID(ctx)
+	if err != nil {
+		panic(err)
```

Comment:
We need to define the revision on the GenesisState so that we avoid this panic on `InitGenesis` (when the revision ID hasn't been set yet)